async.go

```diff
@@ -90,3 +105,52 @@ func panicHandler(ctx context.Context, errCh chan error) {
 func getContextWithTaskName(ctx context.Context, taskName string) context.Context {
 	return loggers.AddToLogContext(ctx, "taskname", taskName)
 }
+
+func GoWithWait(ctx context.Context,
+	taskNames []string,
```

Comment:
passing list of pair of (taskname, task) is much better than two independent list.

Reply:
refactored code to pass task fn and name simple from caller func